pkg/agent/nodeportlocal/portcache/port_table_windows_test.go

```diff
@@ -0,0 +1,179 @@
+//go:build windows
+// +build windows
+
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package portcache
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/tools/cache"
+
+	portcachetesting "antrea.io/antrea/pkg/agent/nodeportlocal/portcache/testing"
+	"antrea.io/antrea/pkg/agent/nodeportlocal/rules"
+	rulestesting "antrea.io/antrea/pkg/agent/nodeportlocal/rules/testing"
+)
+
+const (
+	startPort = 61000
+	endPort   = 65000
+)
+
```

Comment:
Functions like addRuleforFreePort and addRuleForPort needs to called from test file by mocking the top level function and hence internal functions will be considered for percentage UT coverage, this can substantialy increase the coverage, the UT tool does not consider the coverage if the functions are internal, Suggest to follow this approach to increase UT coverage.